resources/config_schema.json

```diff
@@ -0,0 +1,990 @@
+{
```

Comment:
I cannot comment on this yet because I am not familiar yet with the way simulations in Sedtrails are defined.